packages/openapi-generator/src/options/options-per-service.spec.ts

```diff
@@ -1,3 +1,5 @@
+import {ParsedGeneratorOptions} from "./options";
```

Comment:
[req] This needs to be formatted
```suggestion
import { ParsedGeneratorOptions } from "./options";
```